run_classifier.py

```diff
@@ -657,7 +661,7 @@ def tpu_scaffold():
           return tf.train.Scaffold()
 
         scaffold_fn = tpu_scaffold
-      else:
+      elif not FLAGS.do_serve:
```

Comment:
.

Reply:
I realized that I should parameterize that variable to remove FLAGS dependency in function